src/main/java/Miku.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Scanner;
+
+public class Miku {
+    public static void main(String[] args) {
+        String line;
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("______________________");
+        System.out.println("Hello! I'm Miku!\n" + "What can I do for you?");
+        System.out.println("______________________");
+
+        line = in.nextLine();
+        String newItem = line;
+        Task[] storedList = new Task[100];
+        int numberOfListItems = 0;
+
+        while (!line.equals("bye")) {
+            System.out.println("______________________");
+
+            if (line.equals("list")) {
+                System.out.println("Here are ycxdfqwert1e123our list items!");
+                for (int i = 0; i < numberOfListItems; i++) {
```

Comment:
Can maybe think of a simpler way to implement the conditionals instead of have if and for nested in while! :)